infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuInterClusterReplicationServer.java

```diff
@@ -73,8 +73,8 @@ public class CorfuInterClusterReplicationServer implements Runnable {
                     + "[-k <seqcache>] [-T <threads>] [-B <size>] [-i <channel-implementation>] "
                     + "[-H <seconds>] [-I <cluster-id>] [-x <ciphers>] [-z <tls-protocols>]] "
                     + "[--disable-cert-expiry-check-file=<file_path>]"
-                    + "[--metrics]"
-                    + "[-P <prefix>] [-R <retention>] <port> <corfu-server-port>\n"
+                    + "[--metrics] [--corfu-port-for-lr=<corfu-port-for-lr>]"
```

Comment:
same applies here

Reply:
this param comes from the CorfuServer params so need to keep them the same...